zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/configuration/ClusterCfg.java

```diff
@@ -210,6 +224,13 @@ public void setConfigManager(final ConfigManagerCfg configManagerCfg) {
     configManager = configManagerCfg;
   }
 
+  public String initialContactPointsValid() {
```

Comment:
🙃 method name is misleading, it returns an error message if it's not valid. I can't think of a better clean method name though, perhaps it would be better to just inline this validation (and the error message) in `SystemContext`, like we do for the other validations?

Reply:
Yeah I couldn't find any better name to be honest, which is probably a sign :smile: 
I'll inline it with the other :+1: